.github/workflows/ci.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       - name: Download win32 Erlang
```

Comment:
@dgud, the Erlang VM isn't a real 64bits app?

Reply:
It is win32 is the API name on windows.  Have nothing todo (today) with the number of bits on the processer.